lib/files/downloader.ts

```diff
@@ -24,20 +24,17 @@ export class Downloader {
    * @param outputDir The directory where files are downloaded and stored.
    * @param contentLength The content length of the existing file.
    * @param opt_proxy The proxy for downloading files.
-   * @param opt_ignoreSSL Should the downloader ignore SSL.
    * @param opt_callback Callback method to be executed after the file is downloaded.
    * @returns Promise<boolean> Resolves true = downloaded. Resolves false = not downloaded.
    *          Rejected with an error.
    */
   static getFile(
       binary: Binary, fileUrl: string, fileName: string, outputDir: string, contentLength: number,
-      opt_proxy?: string, opt_ignoreSSL?: boolean, callback?: Function): Promise<boolean> {
+      opt_proxy?: string, callback?: Function): Promise<boolean> {
```

Comment:
removed `opt_proxy` param, and originally it's only used for logging, those lines are also removed.